app/models/participant_declaration.rb

```diff
@@ -1,16 +1,17 @@
 # frozen_string_literal: true
 
 class ParticipantDeclaration < ApplicationRecord
+  has_one :profile_declaration
   belongs_to :lead_provider
-  belongs_to :early_career_teacher_profile
+  belongs_to :user
```

Comment:
Do we care about any live data for this change?

Reply:
No. We drop all of the existing ones in the migration before changing the shape. It's all scratch data at this point anyway.